packages/plugin/src/protobufts-plugin.ts

```diff
@@ -62,6 +62,11 @@ export class ProtobuftsPlugin extends PluginBase<OutFile> {
                 "with our option (ts.exclude_options). Set this option if you are certain you do not want \n" +
                 "to include any options at all.",
         },
+        keep_enum_prefix: {
+            description: "By default, if all enum values share a prefix that corresponds with the enum's name, \n" +
+                "the prefix is dropped from the value names. Set this option if you want to disable this behavior \n" +
+                "and keep the prefix intact.",
+        },
```

Comment:
Do we keep the prefix intact, or the enum value name? Suggest to keep it simple:

```suggestion
        keep_enum_prefix: {
            description: "By default, if all enum values share a prefix that corresponds with the enum's name, \n" +
                "the prefix is dropped from the value names. Set this option to disable this behavior.",
        },
```